ComposeSample/.gitignore

```diff
@@ -0,0 +1,45 @@
+### Android template
```

Comment:
@CesarValiente Thanks for your review. I have some small questions here. Do we still need this file here?

Reply:
Yes, I believe you need it since under `ComposeSample` the IDE will generate it's `.idea` folder as well + others that you want to ignore.